pub/refget-openapi.yaml

```
@@ -166,8 +154,15 @@ components:
               type: string
               description: >-
                 Truncated, to 48 characters, SHA-512 checksum of the reference
-                sequence encoded as a HEX string
+                sequence encoded as a HEX string. No longer a preferred serialisation of the SHA-512
               example: 959cb1883fc1ca9ae1394ceb475a356ead1ecceff5824ae7
+            ga4gh:
+              type: string
+              description: >-
+                A ga4gh identifier used to identify the sequence. This is a SHA-512 base64 url encoded
```

To be easily understood, this should be referred to as “base64url-encoded” with a link to RFC 4648 §5.

```suggestion
                A ga4gh identifier used to identify the sequence. This is a base64url-encoded
```

---

Done